doc/how-to/how-to-identify-missing-files.md

```diff
@@ -0,0 +1,47 @@
+# How to identify and fix missing files
+
+## Context
+PlanX allows applicants to upload images alongside their applications. These files are stored on AWS S3. When each file is uploaded, we use Scanii (link) to scan the uploaded files for - 
```

Comment:
is `(link)` here mean you wanted to add a [link](https://scanii.com/), or it's intended?

Reply:
Yep - good catch - meant to come back and add a link to Scanii.com here! Now fixed 👍